nano/secure/account_iterator.hpp

```diff
@@ -0,0 +1,32 @@
+#pragma once
```

Comment:
This file could do with some comments.
How is the end of a sequence detected?
What does the second constructor do?
What is a view expected to be?
A quick example would be nice